lib/core/api/clients/activiti/activiti-client.module.ts

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-import { AboutApi, SystemPropertiesApi } from '@alfresco/js-api';
+import { AboutApi, SystemPropertiesApi, UserProfileApi } from '@alfresco/js-api';
```

Comment:
If I understood correctly, the ultimate goal is to get rid of the js-api dependency from the core? Is this going to be done in the next step for this module here (or maybe whole module moved somewhere else?)

Reply:
Yeah, for now is ok to get it from the `js-api`, we'll get rid of it in the next step